_sass/base/blocks/_404.scss

```diff
@@ -0,0 +1,14 @@
+.content-column {
+  @include span-columns(12);
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets